lib/ReactViews/Tools/DiffTool/DiffTool.tsx

```diff
@@ -48,725 +45,637 @@ import Select from "../../../Styled/Select";
-                `}
-              >
-                <TextSpan medium>
-                  {t("diffTool.labels.saveToWorkbench")}
-                </TextSpan>
-              </Button>
-            </Box>
-          )}
-          {!isShowingDiff && (
-            <LocationPicker
-              terria={terria}
-              location={this.location}
-              onPicking={this.onUserPickingLocation}
-              onPicked={this.onUserPickLocation}
-            />
-          )}
-          {!isShowingDiff &&
-            ReactDOM.createPortal(
-              // Bottom Panel
-              <Box centered fullWidth flexWrap backgroundColor={theme.dark}>
-                <DatePicker
-                  heading={t("diffTool.labels.dateComparisonA")}
-                  item={this.props.leftItem}
-                  externalOpenButton={this.openLeftDatePickerButton}
-                  onDateSet={() => this.showItem(this.props.leftItem)}
-                />
-                <AreaFilterSelection
-                  t={t}
-                  location={this.location}
-                  isPickingNewLocation={this._isPickingNewLocation}
-                />
-                <DatePicker
-                  heading={t("diffTool.labels.dateComparisonB")}
-                  item={this.props.rightItem}
-                  externalOpenButton={this.openRightDatePickerButton}
-                  onDateSet={() => this.showItem(this.props.rightItem)}
-                />
-              </Box>,
-              document.getElementById("TJS-BottomDockLastPortal")!
+                </div>
+                <Text
+                  small
+                  light
+                  textLight
+                  id="TJSDifferenceDisabledButtonPrompt"
+                >
+                  {t("diffTool.labels.disabledButtonPrompt")}
+                </Text>
+              </div>
             )}
-        </div>
-      </WorkflowPanel>
-    );
-  }
-}
+          </div>
+        )}
+        {isShowingDiff && (
+          <Box centered left>
+            <BackButton
+              css={`
+                color: ${theme.textLight};
+                border-color: ${theme.textLight};
+              `}
+              transparentBg
+              onClick={resetTool}
+            >
+              <BoxSpan centered>
+                <StyledIcon
+                  css="transform:rotate(90deg);"
+                  light
+                  styledWidth="16px"
+                  glyph={GLYPHS.arrowDown}
+                />
+                <TextSpan medium>{t("general.back")}</TextSpan>
+              </BoxSpan>
+            </BackButton>
+            <Button
+              primary
+              onClick={closePanel}
+              css={`
+                flex-grow: 1;
+                margin-left: 10px;
+              `}
+            >
+              <TextSpan medium>{t("diffTool.labels.saveToWorkbench")}</TextSpan>
+            </Button>
+          </Box>
+        )}
+        {!isShowingDiff && (
```

Comment:
Can these repeated tests for `!isShowingDiff` be eliminated, or can `isShowingDiff` change value between one test and the next?